src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -517,15 +579,26 @@ export default class MissionWindow extends Component<ThemeProps, State> {
           <MissionOptions title={title} missionNames={missionNames} options={options} />
         </div>
         <div className="geofenceContainer">
-          <h1>Geofencing</h1>
+          <h1>Keep-IN</h1>
```

Comment:
I prefer to see `Geofencing Keep-In` instead of `Keep-IN`, easier to understand what the fields are for